### PR TITLE
add fix to java 8 break in gradle build

### PR DESCRIPTION
**Pull request by jirungaray**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,6 +27,15 @@ allprojects {
  */
 allprojects { project ->
 
+    //Skip Javadoc generation for Java 1.8 as it breaks build
+    if (JavaVersion.current().isJava8Compatible()) {
+        allprojects {
```
Comment (samtstern) [inline build.gradle]: Since this references allprojects, I think it should be outside of the outer allprojects block rather than nested.

**Comment by jirungaray** [comment]
I signed it!

File: build.gradle (hunk 1 above)
Comment (jirungaray) [inline build.gradle]: Agreed. Any convention/position in  particular this check should be placed?

**Comment by samtstern** [comment]
@jirungaray thanks, merging!

**Pull request merged**